### PR TITLE
Fix for AKS Kubelet Servicemonitor

### PR DESCRIPTION
**Pull request by hisarbalik**
Kubelet service monitor for prometheus is down on AKS cluster, service monitor monitoring wrong port. Kubelet not working anymore on http port, this fix will switch to the https metric port.

### CODE REVIEW

File: resources/monitoring/templates/kyma-additions/aks-kubelet-monitoring-patch.yaml [hunk 1]
```diff
@@ -25,20 +25,30 @@ data:
     metadata:
       labels:
         app: {{ template "prometheus-operator.name" . }}-kubelet
-        {{- include "prometheus-operator.labels" . | indent 8 }}
+    {{- include "prometheus-operator.labels" . | indent 8 }}
```
Comment (rakesh-garimella) [inline resources/monitoring/templates/kyma-additions/aks-kubelet-monitoring-patch.yaml:28]: i think this should be indented 2 spaces.

Comment (hisarbalik) [reply resources/monitoring/templates/kyma-additions/aks-kubelet-monitoring-patch.yaml]: fixed

File: resources/monitoring/templates/kyma-additions/aks-kubelet-monitoring-patch.yaml [hunk 2]
```diff
@@ -32,13 +32,27 @@ data:
       endpoints:
         - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
           honorLabels: true
-          port: http-metrics
-          scheme: http
+          port: https-metrics
```
Comment (a-thaler) [inline resources/monitoring/templates/kyma-additions/aks-kubelet-monitoring-patch.yaml:35]: After that changes, what is the difference to the original kubelet servicemonitor?

**Comment by hisarbalik** [comment]
> When seeing the changes it look slike that they aligned the aks kubelet to the standard one making the hole custom aks handling obsolete. Am I right? If yes, the goal must be to remove the custom aks stuff

You are right, patch for AKS removed testet on cluster.

**Comment by a-thaler** [comment]
As the fix seem to stabilize the upgrade job as well, I force merged it in order to bypass the upgrade job fail

**Pull request merged**